Cargo.toml

```diff
@@ -11,4 +11,4 @@ documentation = "http://doc.servo.org/fnv/"
 [lib]
 name = "fnv"
 path = "lib.rs"
-doctest = false
```

Comment:
I use `doctest = false` when there are no doctest. `rustdoc --test` would run and take some time just to find out it has nothing to do.